### PR TITLE
Add env variable

### PR DESCRIPTION
**Pull request by pacospace**
Signed-off-by: Francesco Murdaca <fmurdaca@redhat.com>

## This introduces a breaking change

- [ ] Yes
- [x] No

## This Pull Request implements

Add environment variable to distinguish between metrics in test, stage and prod

### CODE REVIEW

File: thoth/advise_reporter/advise_reporter.py [hunk 1]
```diff
@@ -31,10 +31,12 @@
 _METRIC_ADVISE_TYPE = Gauge(
     "thoth_advise_message_number",
     "Number of thamos advise provided per message.",
-    ["advise_message"],
+    ["advise_message", "thoth_environment"],
     registry=prometheus_registry,
 )
 
+_THOTH_ENVIRONMENT = os.getenv("THOTH_ENVIRONMENT")
```
Comment (fridex) [inline thoth/advise_reporter/advise_reporter.py]: Should this be always set?

```python
_THOTH_ENVIRONMENT = os.environ["THOTH_ENVIRONMENT"]
```

File: openshift/consumer-template.yaml [hunk 2]
```diff
@@ -95,6 +100,8 @@ objects:
                   value: ${{DEBUG_LEVEL}}
                 - name: APP_SCRIPT
                   value: "app.sh"
+                - name: THOTH_ENVIRONMENT
+                  value: "${THOTH_ENVIRONMENT}"
```
Comment (fridex) [inline openshift/consumer-template.yaml]: We have ``THOTH_DEPLOYMENT_NAME`` that has slightly different semantics but captures test/stage/prod with cluster name. Would it be more suitable?

File: thoth/advise_reporter/advise_reporter.py (hunk 1 above)
Comment (pacospace) [reply thoth/advise_reporter/advise_reporter.py]: Yes, Thanks!

File: openshift/consumer-template.yaml (hunk 2 above)
Comment (pacospace) [reply openshift/consumer-template.yaml]: Yes in this case that is reusable actually, thanks!

**Review by goern — APPROVED** [review]

**Pull request merged**